pkg/controller/applicationmonitoring/templateHelper.go

```diff
@@ -96,6 +99,8 @@ type Parameters struct {
 	PrometheusStorageRequest         string
 	PrometheusInstanceNamespaces     string
 	AlertmanagerInstanceNamespaces   string
+	Affinity                         string
+	ZoneTopologyLabel                string
```

Comment:
```suggestion
```
I don't see this variable is used anywhere. I guess it was from the previous solution